src/shared/modules/guides/guidesDuck.ts

```diff
@@ -72,6 +86,32 @@ interface AddGuideAction {
   guide: Guide
 }
 
+export const fetchRemoteGuideEpic: Epic<
+  FetchGuideAction | SetGuideAction | OpenAction | AddGuideAction,
+  GlobalState
+> = (action$, store$) =>
+  action$.ofType(FETCH_GUIDE).flatMap(action => {
+    const initialSlide = tryGetRemoteInitialSlideFromUrl(
+      (<FetchGuideAction>action).identifier
+    )
+    return Observable.fromPromise(
+      resolveGuide((<FetchGuideAction>action).identifier, store$.getState())
+    ).mergeMap(({ title, identifier, slides, isError }) => {
+      const guide: Guide = {
```

Comment:
I've also been thinking a bit about if handling this with redux observable is the best way to go about things.. What we could do is this: store guides to localstorage (except the slides). Then when we open a guide that has slides null we treat that as if the slides are "loading", then we call the resolveGuide method updating the slide in redux and voloua!

Reply:
It's get rid of a lot of moving parts, let's discuss later

Reply:
@OskarDamkjaer - as discussed offline, we will leave the behaviour for now. Better to have an improved behaviour in the future.